subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/logcat/ResultLogcatAccessor.kt

```diff
@@ -0,0 +1,13 @@
+package com.avito.instrumentation.internal.logcat
+
+import com.avito.android.Result
+
+internal class ResultLogcatAccessor(val result: Result<String>) : LogcatAccessor {
+
+    override val isAvailable: Boolean
+        get() = result.isSuccess()
+
+    override fun getLogs(): Logcat {
+        return result.map { Logcat(it) }.getOrElse { Logcat.STUB }
+    }
```

Comment:
Example for https://github.com/avito-tech/avito-android/pull/1036/files#r643426727

I will invoke `getLogs()` and get empty string.

Reply:
https://github.com/avito-tech/avito-android/pull/1036#discussion_r643462103